.github/workflows/tools.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/guid_generator
     - name:  GUID Generator Tests
-      if: steps.changes.outputs.guid_generator == 'true'
+      if: steps.changes.outputs.guid_generator == 'true' OR steps.changes.outputs.instance_validator == 'true'
```

Comment:
the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

Reply:
Done.